src/noc/containers/noc-field/postcode/noc-postcode-field.html

```diff
@@ -0,0 +1,10 @@
+<div class="govuk-form-group" [ngClass]="{'form-group-error': !postcodeControl.valid}">
```

Comment:
This looks virtually identical to `noc-phone-uk-field.html` and `not-text-field.html`, just parameterised. Wouldn't it be better to have a single template with appropriate parameters? In these cases, the only difference can actually see is the `type` attribute on the `<input />` element - all other variations come down to a naming convention within the `.component.ts` file.

Also, why is this not a `.component.html` file?

Reply:
Agreed, but I would keep the same naming convention and template model to align with ccd toolkit.

Reply:
Which bit is aligned with the CCD toolkit? Just the naming convention of not having `.component.html`?

Reply:
the original design is from 
ccd-case-ui-toolkit\src\shared\components\palette
I think we should keep this field/question type separately but not a generic component even they are all text.

Reply:
I disagree.

If there's a good reason to do our own version of a component (I assume there is or you wouldn't have done all this work), then there's an equally good reason to make our own version better than the original. Duplication is never a good thing as it makes the code harder to maintain (and can needlessly inflate the size of the bundle.)

Reply:
.component.html are now attached to the file